src/main/java/org/fogbowcloud/arrebol/execution/TaskExecutionResult.java

```diff
@@ -1,47 +1,39 @@
 package org.fogbowcloud.arrebol.execution;
 
 import org.fogbowcloud.arrebol.models.command.Command;
-
 import java.util.Arrays;
 
 public class TaskExecutionResult {
 
-    //this class is more like an struct, not very much OO but it ok
+    // this class is more like an struct, not very much OO but it ok
```

Comment:
there a few typos in this class (e.g but it ok ...). I guess I made these mistakes. Could you correct my errors?

Reply:
Done.